db/migrate/20230221082102_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+
+      t.timestamps
```

Comment:
- [ ] Please 🙏🏽 add the references to the user and the post tables. It represents the relationship between these tables and the comments tables.